handbook/rbac/rbac.md

```diff
@@ -0,0 +1,128 @@
+# Workspaces and Role-based Access Control
```

Comment:
With the exception of the Automatic Data Encapsulation section and parts of the Users section, SPD covers most of these additions. I'm concerned about duplication of content and labor. If our strategy about what goes into the ops handbook vs. SPD has changed, let me know so I can plan accordingly.